### PR TITLE
Update tag_residential_loadshifting_assets.yaml

### PR DESCRIPTION
**Pull request by ryan-oreilly**
Add Electric Vehicle_55 which represents expectations for EV energy demand using the Fit for 55 scenario

### CODE REVIEW

**Review by sandrinecharousset — CHANGES_REQUESTED** [review]

File: definitions/variable/tag_residential_loadshifting_assets.yaml [hunk 1]
```diff
@@ -6,6 +6,8 @@
 - Residential Load Shifting Asset:
   - Electric Vehicle:
       description: electric vehicles
+   - Electric Vehicle_55:
```
[the remark this answers is not in the document]
Comment (danielhuppmann) [reply definitions/variable/tag_residential_loadshifting_assets.yaml:9]: Yes, I agree - this is not the intended usage of a variable. Please remove or close this PR and start a new one that has only the changes to the other descriptions.

[the remark this answers is not in the document]
Comment (sandrinecharousset) [reply definitions/variable/tag_residential_loadshifting_assets.yaml:9]: @ryan-oreilly

File: definitions/variable/tag_residential_loadshifting_assets.yaml [hunk 2]
```diff
@@ -6,6 +6,8 @@
...
+      description: electric vehicles using Fit for 55 scenario
```
Comment (sandrinecharousset) [inline definitions/variable/tag_residential_loadshifting_assets.yaml:10]:
```suggestion
    
```

File: definitions/variable/tag_residential_loadshifting_assets.yaml (hunk 1 above)
Comment (sandrinecharousset) [inline definitions/variable/tag_residential_loadshifting_assets.yaml:9]:
```suggestion
   
```

**Review by sandrinecharousset — CHANGES_REQUESTED** [review]
suggestion to remove electric vehicle_55

**Review by sandrinecharousset — APPROVED** [review]

**Pull request merged**